src/components/calcite-action/calcite-action.scss

```diff
@@ -64,16 +64,31 @@
   }
 }
 
-:host([scale="s"]) .button {
-  @apply p-2;
+:host([scale="s"]) {
+  .button {
+    @apply p-2 text--2h;
```

Comment:
We should set the font-weight to "normal".
"medium" seems pretty heavy here.

Reply:
Do you figure this would be good only on "small" scale or for all instances of button?

Reply:
@asangma Updated the small scale only on this one.

Reply:
Sorry. I meant for all scales.

Reply:
Sounds good. Will make the correction.

Reply:
@asangma Updated for all scales.